js/page-specific/alerts/alerts-display.js

```diff
@@ -0,0 +1,47 @@
+namespacer('bcpl.pageSpecific.alerts');
+
+bcpl.pageSpecific.alerts.alertDisplay = (($, Handlebars, moment, CONSTANTS) => {
+	const alertsTemplateSelector = '#alerts-handlebars-template';
+	const alertsTargetSelector = '#alerts-handlebars-target';
+	const dateFormat = 'M/D/YYYY';
+
+	const render = (alerts) => {
+		const alertsTemplateHtml = $(alertsTemplateSelector).html();
+
+		if (alertsTemplateHtml && alertsTemplateHtml.length) {
+			const alertsTemplate = Handlebars.compile(alertsTemplateHtml);
+			const renderedHtml = alertsTemplate(alerts);
+			$(alertsTargetSelector).html(renderedHtml);
+		}
+	};
+
+	const getAlertData = (callback) => {
+		$.ajax(CONSTANTS.baseApiUrl + CONSTANTS.shared.urls.alerts)
+			.then(
+				alerts => onAlertsSuccess(alerts, callback),
+				error => console.error(error)
```

Comment:
this could actually just be

```javascript
...
(alerts => onAlertsSuccess(alerts, callback),
console.error)
```

however, do we actaully want this to right to the log?

Reply:
*write

Until we have a reporting system in place, yeah, unfortunately.